pytest/tests/loadtest/locust/locustfiles/linkdrop.py

```diff
@@ -0,0 +1,64 @@
+"""
+A workload with Linkdrop operations.
+"""
+
+import logging
+import pathlib
+import random
+import sys
+import ed25519
+import base58
+import string
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / 'lib'))
+
+import key
+from configured_logger import new_logger
+from locust import constant_throughput, task
+from common.base import Account, Deploy, NearNodeProxy, NearUser, FunctionCall, INIT_DONE
+from common.linkdrop import LinkdropContract, AddKey, ClaimDrop
+
+logger = new_logger(level=logging.WARN)
+
+
+class LinkdropUser(NearUser):
+    """
+    Funder creates a drop on Keypom contract. Add keys to the drop.
+    Claimer claims the drop.
+    """
+    # Only simple claims right now. Do FT, NFT etc later.
+
+    # Each Locust user will try to send one transaction per second.
+    # See https://docs.locust.io/en/stable/api.html#locust.wait_time.constant_throughput.
+    wait_time = constant_throughput(1.0)
+
+    @task
+    def create_and_claim_drop(self):
+        # Generate a random key pair
+        keypair = key.Key.implicit_account()
+        public_keys = []
```

Comment:
Nit: can just define this as `public_keys = [keypair.pk]`

Reply:
Or maybe even inlining it in `AddKey` constructor, as this variable it not used anywhere else.